docs/content/docs/ops/state/checkpoints_vs_savepoints.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Checkpoints VS Savepoints"
+weight: 10
+type: docs
+aliases:
+  - /ops/state/checkpoints_vs_savepoints.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Checkpoints VS Savepoints
+
+## Overview
+
+Conceptually, Flink's [savepoints]({{< ref "docs/ops/state/savepoints" >}}) are different from [checkpoints]({{< ref "docs/ops/state/checkpoints" >}}) 
+in a way that's analogous to how backups are different from recovery logs in traditional database systems.
+
+The primary purpose of checkpoints is to provide a recovery mechanism in case of unexpected job failures. 
+A [checkpoint's lifecycle]({{< ref "docs/dev/datastream/fault-tolerance/checkpointing" >}}) is managed by Flink, 
+i.e. a checkpoint is created, owned, and released by Flink - without user interaction. 
+Because checkpoints are being triggered often, and are relied upon for failure recovery, the two main design goals for the checkpoint implementation are 
+i) being as lightweight to create and ii) being as fast to restore from as possible. 
+Optimizations towards those goals can exploit certain properties, e.g., that the job code doesn't change between the execution attempts. 
+Checkpoints are usually deleted after the job has been terminated by the user (except if explicitly configured as retained checkpoints).
```

Comment:
This sentence is redundant with the line below; seems like we don't need it twice.

```suggestion
```